tests/index.js

```diff
@@ -611,6 +611,14 @@ t('Transform nested json in arrays', async() => {
   return ['aBcD', (await sql`select '[{"a_b":1},{"c_d":2}]'::jsonb as x`)[0].x.map(Object.keys).join('')]
 })
 
+t('Transform null json in arrays', async() => {
+  const sql = postgres({
+    ...options,
+    transform: postgres.camel
+  })
+  return [null, (await sql`select '[{"a_b":null},{"c_d":null}]'::jsonb as x`)[0].x.map(Object.keys).join('')]
```

Comment:
I'm not sure this condition does what you think it does (I imagine this test will fail)

this `.map(Object.keys)` and `.join()` don't seem like the right thing to do here in this test - you don't care about the object keys, you care about the values